### PR TITLE
SubscribeHandler not working with NonVerbose mode

### PR DESCRIPTION
**Pull request by kevin96666**

### CODE REVIEW

File: recws.go [hunk 1]
```diff
@@ -404,25 +404,27 @@ func (rc *RecConn) connect() {
 		rc.httpResp = httpResp
 		rc.mu.Unlock()
 
+
```
Comment (loeffel-io) [inline recws.go]: Please remove this whiteline

File: recws.go [hunk 2]
```diff
@@ -404,25 +404,27 @@ func (rc *RecConn) connect() {
...
+			}
 
-				if !rc.hasSubscribeHandler() {
-					return
-				}
-
+			if rc.hasSubscribeHandler() {
 				if err := rc.SubscribeHandler(); err != nil {
-					log.Fatalf("Dial: connect handler failed with %s", err.Error())
+					if !rc.getNonVerbose() {
```
Comment (loeffel-io) [inline recws.go]: here we should not hide the log.Fatal error message 
please just remove the getNonVerbose logic at this point

**Comment by loeffel-io** [comment]
close #22

File: recws.go (hunk 2 above)
Comment (kevin96666) [reply recws.go]: True, I guess I was tired

File: recws.go (hunk 1 above)
Comment (kevin96666) [reply recws.go]: done

**Pull request merged**